_posts/2017-04-18-release-0.3p2.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Eve Alpha v0.3 preview 2"
+author: "Corey Montella"
+tags: []
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://play.witheve.com))_
+
+Now that we've got the v0.3 preview out the door, we're ramping up to a more regular release schedule. In that spirit, today we're releasing the second preview of Eve v0.3, which largely improves built in watchers, but contains some other goodies as well. 
+
+### What's new?
+
+Let's take a look at some of the bigger improvements.
+
+#### Embedding Eve Elements in Your App
+
+You can now embed Eve rendered elements into your non-Eve application. This is accomplished with the `addExternalRoot(tag:string, element:HTMLElement)` function, which registers an arbitrary root element to serve as parents to Eve-rendered elements. It works like this:
+
+```javascript
+import {Program} from "witheve";
+
+let program = new Program("my program");
+let htmlWatcher = program.attach("html");
+
+let eveWrapper = document.querySelector("#eve-wrapper");
+htmlWatcher.addExternalRoot("my-root", eveWrapper);
+```
+
+This creates a record tagged "my-root" in Eve, which you can treat as any other Eve element.
```

Comment:
We should make it clear that it's calling `addExternalRoot("my-root", someElement)` that creates it, not the block below. It currently looks like the latter, which is really just a regular block searching for the special external root element we previously created.